basetest.pm

```diff
@@ -392,7 +392,7 @@ sub runtest {
 
     $self->done();
     $self->{execution_time} = execution_time($starttime);
-    bmwqemu::diag(sprintf("||| finished %s %s at %s (%d s)", $name, $self->{category}, POSIX::strftime('%F %T', gmtime), $self->{execution_time}));
+    bmwqemu::diag(sprintf("||| finished %s %s at %s (%d s)", $name, $self->{category}, $self->{execution_time}));
```

Comment:
This change looks wrong, The placeholder string is unchanged (and still contains 4 placeholders) while one of the parameters has been removed. The placeholder for the timestamp should have been removed as well, e.g. `"||| finished %s %s (runtime: %d s)"`. I also added (`runtime: `) because now it isn't that clear anymore what the parenthesis are about.

Reply:
I shortly created a fix: https://github.com/os-autoinst/os-autoinst/pull/1722

Reply:
Good catch.
I wonder why this wasn't catched as a warning:
```
% perl -wE'say sprintf "%s %s", 23'
Missing argument in sprintf at -e line 1.
23
```

Reply:
Good question. Maybe this is only covered by a forked process (which we don't take into account when checking for warnings)?

Reply:
hm, sorry. I thought our tests would have covered that

Reply:
I'm sure the line is invoked somewhere within our tests. Whether we have a check is unfortunately a different story.